assets/scss/components/_avatar.scss

```diff
@@ -148,7 +154,7 @@ $avatarIconSizeProps: (height, line-height, width);
 	text-indent: initial;
 	background-color: darken($avatarBg, 6%);
 
-	.svg {
+	.svg--profile {
```

Comment:
does this still need to be nested under `.avatar--noPhoto`?

Reply:
Unfortunately, yes, until we move to CSS Modules. There's a chance we may use an `Icon shape="profile"` outside of an avatar, and disqualifying these rules from `.avatar--noPhoto` could cause side effects.